src/main/kotlin/org/rust/ide/intentions/RsShowSingleStepMacroExpansionIntention.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.parentOfType
+import org.rust.ide.actions.macroExpansion.MacroExpansionViewCreator
+import org.rust.lang.core.psi.RsMacroCall
+
+class RsShowSingleStepMacroExpansionIntention : RsElementBaseIntentionAction<RsMacroCall>() {
```

Comment:
`RsShowSingleStepMacroExpansionIntention` and `RsShowRecursiveMacroExpansionIntention` look very similar.
Maybe create base class for them to move common code?

Reply:
@Undin ok, will do
Just personally don't like inheritance :sweat_smile: